extensions/amp-image-viewer/0.1/amp-image-viewer.js

```diff
@@ -148,9 +148,12 @@ export class AmpImageViewer extends AMP.BaseElement {
     if (this.loadPromise_) {
       return this.loadPromise_;
     }
+    const laidOutPromise =
+      elementByTag(this.sourceAmpImage_, 'i-amphtml-sizer') !== null
```

Comment:
Ideally, there would be a better way to check if an element is already laid out, but I don't currently know of one.

Reply:
`hasAttribute('i-amphtml-layout')` is safer. I don't think we create sizer for all layouts.

Reply:
Good idea.